prov/gni/test/api.c

```diff
@@ -762,9 +999,11 @@ void do_atomic_write_fetch(void)
 	/* u64 */
 	*((uint64_t *)source) = SOURCE_DATA;
 	*((uint64_t *)target) = TARGET_DATA;
+
 	sz = fi_atomic(ep[0], source, 1,
-		       loc_mr[0], gni_addr[1], (uint64_t)target, mr_key[1],
-		       FI_UINT64, FI_ATOMIC_WRITE, target);
+			   loc_mr[0], gni_addr[1],
```

Comment:
I was changing the lines here and decided I'd modify the indentation to fit the style used in other locations in this file to maintain a uniform convention.